### PR TITLE
[WEB-5569] Send Events from Off-App Tracker Chrome Extension to Lambda

### PR DESCRIPTION
**Pull request by VeranikaVasilchykava**
https://bowery.atlassian.net/browse/WEB-5569

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
         "manifest_version": 2,
-    
+
```
Comment (MakaryHryshkin) [inline src/manifest.json:3]: empty line?

File: src/manifest.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
         "description": "Track and limit time your activity in the browser every day.",
 
         "options_page": "options.html",
-    
+
```
Comment (MakaryHryshkin) [inline src/manifest.json:12]: same

File: src/scripts/background.js [hunk 3]
```diff
@@ -250,18 +265,168 @@ function setDefaultValueForNewSettings() {
...
+            lastActiveTabUrl = tab.url;
+            tabToUrl[tabId] = tab.url;
+        }
+    });
+
+    chrome.tabs.onRemoved.addListener(async (tabId, removeInfo) => {
+        if (removeInfo.isWindowClosing) {
+            return;
+        }
+        if ( tabToUrl[tabId] !== lastActiveTabUrl) {
```
Comment (MakaryHryshkin) [inline src/scripts/background.js]: redundant space?

**Review by MakaryHryshkin — APPROVED** [review]

File: src/manifest.json (hunk 2 above)
Comment (VeranikaVasilchykava) [reply src/manifest.json:12]: I didn't add any empty lines intentionally in manifest.json. All these lines are already present in master-branch.

File: src/scripts/background.js (hunk 3 above)
Comment (VeranikaVasilchykava) [reply src/scripts/background.js]: yep, didn't notice. I'll remove it with final commit

**Review by agrren — APPROVED** [review]

**Review by igor-kazakevich — APPROVED** [review]
Nice work!

**Review by igor-kazakevich — APPROVED** [review]

**Review by MakaryHryshkin — APPROVED** [review]
lgtm

**Review by mrromro — APPROVED** [review]

**Review by MakaryHryshkin — APPROVED** [review]

**Pull request merged**